zenoh/src/net/runtime/adminspace.rs

```diff
@@ -65,6 +66,7 @@ enum PluginDiff {
 impl AdminSpace {
     pub async fn start(runtime: &Runtime, plugins_mgr: plugins::PluginsManager, version: String) {
         let zid_str = runtime.zid.to_string();
+        let metadata = runtime.metadata.to_string();
```

Comment:
Not sure about this.

I quickly tested it and it returns a string, not a JSON object.

I start the router with this configuration:
```
{
	"metadata":{
		"dns":"myprecious.zettascale.cloud",
		"project_id":"1231"
	},
	"id":"123123",
	"plugins":{
		"rest":{
		"__required__": true,
		"http_port":8000
		}
	}
}
```
Then fetch its configuration from the Admin Space: `curl -X GET http://127.0.0.1:8000/@/router/123123 | jq '.[0].value.metadata' ` 

And I get this: `"{\"dns\":\"myprecious.zettascale.cloud\",\"project_id\":\"1231\"}"`

While I was expecting this: `{"dns":"myprecious.zettascale.cloud","project_id":"1231"}`


We can also see that it is a string and not an object when running: `curl -X GET http://127.0.0.1:8000/@/router/123123 | jq`

that gives:

```
[
  {
    "key": "@/router/123123",
    "value": {
      "locators": [
        "tcp/[fdda:399e:8be6::2f1]:7447",
        "tcp/[fdda:399e:8be6:0:7e51:f075:e1f:2d83]:7447",
        "tcp/[fe80::50b4:a759:a903:d88e]:7447",
        "tcp/[fe80::42:d7ff:fe9a:fcc0]:7447",
        "tcp/[fe80::8cd0:abff:fe0c:5d9e]:7447",
        "tcp/[fe80::645f:7eff:fed7:97f5]:7447",
        "tcp/[fe80::38a4:70ff:fe3d:7ddd]:7447",
        "tcp/[fe80::84ab:25ff:fe66:7504]:7447",
        "tcp/[fe80::a8df:98ff:fee0:a1a5]:7447",
        "tcp/[fe80::ec8c:5dff:fea8:60e6]:7447",
        "tcp/192.168.86.57:7447",
        "tcp/172.17.0.1:7447",
        "tcp/172.18.0.1:7447"
      ],
      "metadata": "{\"dns\":\"myprecious.zettascale.cloud\",\"project_id\":\"1231\"}",
      "plugins": {
        "rest": {
          "path": "/home/gabrik/Workspace/zettascale/zenoh/zenoh/target/debug/libzenoh_plugin_rest.so"
        }
      },
      "sessions": [],
      "version": "v0.10.0-dev-19-g74c4633d built with rustc 1.71.0 (8ede3aae2 2023-07-12)",
      "zid": "123123"
    },
    "encoding": "application/json",
    "time": "None"
  }
]
```

So maybe it should be a `serde_json::Value` also here.

Reply:
ooops. Fixed